src/stores/appcatalog/actions.ts

```diff
@@ -85,7 +83,7 @@ export const prepareIngressTabData = createAsynchronousAction<
 
       await Promise.all([
         dispatch(loadClusterApps({ clusterId: payload.clusterId })),
-        catalogLoadIndex(gsCatalog)(dispatch, () => state),
+        dispatch(catalogLoadIndex(gsCatalog)),
```

Comment:
Any reason for not using `dispatch` here?

Reply:
Very good question.. I copied this part from the initial implementation I did in the component. :shrug: